apps/marginfi-v2-ui/src/utils/mrgnActions.ts

```diff
@@ -879,6 +879,7 @@ export async function mintLstToken({
 // Helpers //
 // ------------------------------------------------------------------//
 
+// PDA not being able to create a bank?
```

Comment:
what do you mean with this?

Reply:
this was a comment i made while debugging a SquadsX issue. it was a note to myself and forgot to delete it. i was mistaken in thinking at our code wasn't allowing off-curve signers to create banks